lib/halfnarp.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
```

Comment:
I personally don't have the 32c3 data but I guessed it should still be around.
Well why not call `rake halfnarp:import 33c3 https://…/votes.json`

This way at least other people (like me) could import data for another event later, without changing the code on the production system or in the repo.